tensorflow_model_analysis/metrics/metric_specs.py

```diff
@@ -323,8 +323,9 @@ def default_multi_class_classification_specs(
           tf.keras.metrics.Precision(name='precision', top_k=top_k),
           tf.keras.metrics.Recall(name='recall', top_k=top_k)
       ])
-    binarize = config.BinarizationOptions().CopyFrom(binarize)
-    binarize.ClearField('top_k_list')  # pytype: disable=attribute-error
+    _binarize = config.BinarizationOptions()
+    _binarize.CopyFrom(binarize)
```

Comment:
If use this approach, then would need to do the following:

if binarize is not None:
   ...
else:
  _binarize = binarize


However, since the CLA is failing, I might just send a separate commit if that is ok with you.

Reply:
Ahh derp, good catch! Well, darn. I was hoping my org would move faster on the CLA but don't let me hold up dev!